packages/cli/src/validators/createTableDefOverriderList.ts

```diff
@@ -0,0 +1,36 @@
+import { ColumnDefProperties } from '@synthql/queries';
+import { SchemaDefOverrides } from '../types/CliConfig';
+
+interface TableDefOverrider {
+    isValidQualifiedTableName: (qualifiedTableName: string) => boolean;
+    applyColumnDefOverride: (
+        columnName: string,
+    ) => Partial<ColumnDefProperties> | undefined;
+}
+
+type TableDefOverriderList = Array<TableDefOverrider>;
```

Comment:
Another example of bad naming. The name is worse than the thing being named. This is just a layer of indirection.